<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <title>Computational analysis of protein-protein interactions by hdinkel</title>

    <link rel="stylesheet" href="stylesheets/styles.css">
    <link rel="stylesheet" href="stylesheets/github-light.css">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <h1>Computational analysis of protein-protein interactions</h1>
        <p>Course material for &quot;Computational analysis of protein-protein interactions: Sequences, networks and diseases&quot;</p>

        <p class="view"><a href="https://github.com/hdinkel/ppisnd">View the Project on GitHub <small>hdinkel/ppisnd</small></a></p>


        <ul>
          <li><a href="https://github.com/hdinkel/ppisnd/zipball/master">Download <strong>ZIP File</strong></a></li>
          <li><a href="https://github.com/hdinkel/ppisnd/tarball/master">Download <strong>TAR Ball</strong></a></li>
          <li><a href="https://github.com/hdinkel/ppisnd">View On <strong>GitHub</strong></a></li>
        </ul>
      </header>

      <section>

        <div style="font-size:24pt;">Visit the <a href="http://events.embo.org/16-protein-protein/">course website</a></div>
          <div style="font-size:24pt;">View the <a href="programme">Programme</a></div>
<h3>About this practical course</h3>

Recent growth in protein-protein interaction (PPI) data provides new opportunities for gaining insights into biological systems. A major focus of this EMBO Practical Course will be teaching the importance of PPIs in these systems, and how bioinformatics tools can be harnessed for understanding and controlling physiological states and when these states are perturbed by disease.
</br>
Ideal participants for this course are bench scientist who are working to obtain data related to protein-protein interactions, or already have such data they are working to analyse. This might include 3D protein structural data, results from low- or high-throughput PPI detection methods (yeast-two-hybrid, TAP-tagged pulldowns, etc.), datasets for building biological networks that could be relevant for inferring PPIs (e.g. gene expression data) or predicting potential interaction partners for proteins or systems of interest.
</br>
This EMBO Practical Course aims to train participants to use bioinformatics tools for predicting and analysing PPIs from their own datasets. Without these skills, bench scientists risk missing important insights from such data. Another aim is to give trainees direct access to trainers who are developers of key tools (Pfam, ELM, IUPRED, Cytoscape, STRING, CHIMERA).
</br>
The content of the course will be similar to that of an EMBO Practical Course taught in Norwich in September 2015. These pages contain all the material taught during the course, you may find it useful to look at this when deciding whether to apply for the course.
</br>
We initially focus on predicting PPI modules from protein sequences, highlighting differences (structure, function, and bioinformatics) between globular protein "domains" and non-globular intrinsically-disordered peptide regions, the latter being rich in short protein-binding regions involved in signal transduction and other functions. Understanding the differing roles these modules play in PPIs is key to unraveling the dynamism and complexity of cellular systems. The EMBO Practical Course ends by focusing on large PPI datasets, emphasizing a network perspective.

      </section>
      <footer>
        <p>This project is maintained by <a href="https://github.com/hdinkel">hdinkel</a></p>
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="javascripts/scale.fix.js"></script>
    
  </body>
</html>
